pkg/apis/multiclusterdns/doc.go

```diff
@@ -15,11 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-
-
```

Comment:
This is generated by kubebuilder. And all the subsequent changes in doc.go. please exclude them.